cmake/modules/AddCIRCT.cmake

```diff
@@ -11,7 +11,16 @@ function(add_circt_interface interface)
 endfunction()
 
 function(add_circt_doc doc_filename command output_file output_directory)
-  add_mlir_doc(${ARGV})
+  set(LLVM_TARGET_DEFINITIONS ${doc_filename}.td)
+  tablegen(MLIR ${output_file}.md ${command} "-I${MLIR_MAIN_INCLUDE_DIR}" "-I${MLIR_INCLUDE_DIR}")
```

Comment:
Seems like we'll need something like CIRCT_INCLUDE_DIR too?

Reply:
Added, but it was working without that also.